src/pages/index.js

```diff
@@ -116,10 +191,15 @@ import { PopupWithImage } from "../components/PopupWithImage";
   //event listeners
 
   profileEditButton.addEventListener("click", () => {
-    editPopup.setPreviewedValues({
-      name: profileName.innerText,
-      description: profileDescription.innerText,
+    api.getProfileInfo().then((data) => {
```

Comment:
You don't need to call `api.getProfileInfo()` when you open the profile modal because it's a waste of resources on fetching and making the server busy

You have `getUserInfo` for that in `UserInfo`

Reply:
You haven't corrected this remark https://github.com/ObbieOnOblivion/se_project_aroundtheus/pull/1#discussion_r1438599255